tools/azsdk-cli/azsdk.csproj

```diff
@@ -0,0 +1,18 @@
+<Project Sdk="Microsoft.NET.Sdk">
```

Comment:
Did you not want to pull in the existing MCP code we have already?

Reply:
This is just a placeholder to release something to test. It will be entirely replaced.